akka-http/src/main/scala/akka/http/scaladsl/server/EntityStreamingSupport.scala

```diff
@@ -0,0 +1,210 @@
+/*
+ * Copyright (C) 2009-2015 Typesafe Inc. <http://www.typesafe.com>
+ */
+package akka.http.scaladsl.server
+
+import akka.NotUsed
+import akka.event.Logging
+import akka.http.javadsl.{ model ⇒ jm }
+import akka.http.scaladsl.model._
+import akka.stream.scaladsl.{ Flow, Framing }
+import akka.util.ByteString
+
+/**
+ * Entity streaming support, independent of used Json parsing library etc.
```

Comment:
This is independent of JSON. Maybe just point to `JsonEntityStreamingSupport`/`CsvEntityStreamingSupport` as example implementations and get rid of the "JSON" references in this class?